homeassistant/components/alarmdecoder/alarm_control_panel.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, auto_bypass):
         self._ready = None
         self._zone_bypassed = None
         self._auto_bypass = auto_bypass
+        self._code_arm_required = False
```

Comment:
Please make this a config option, see the mqtt alarm control panel as an example.